src/main/config/config_eeprom.c

```diff
@@ -102,6 +106,132 @@ bool loadEEPROMFromExternalFlash(void)
 }
 #endif
 
+#ifdef EEPROM_IN_SDCARD
+
+enum {
+    FILE_STATE_NONE = 0,
+    FILE_STATE_BUSY = 1,
+    FILE_STATE_FAILED,
+    FILE_STATE_COMPLETE,
+};
+
+volatile uint8_t fileState = FILE_STATE_NONE;
```

Comment:
I don't see any reason for volatile (and use of volatile raises 'suspicious' flag) ...

Reply:
thanks, I'll check that.

Reply:
fixed, thanks